diff --git a/node_modules/@debank/rabby-api/dist/index.js b/node_modules/@debank/rabby-api/dist/index.js
index 55a7ee5..0d6ae4d 100644
--- a/node_modules/@debank/rabby-api/dist/index.js
+++ b/node_modules/@debank/rabby-api/dist/index.js
@@ -44,10 +44,12 @@ export class OpenApiService {
                 const { method, url, params } = genSignParams(config);
                 const res = sign.cattleGsW(params, method, url);
                 config.headers = config.headers || {};
-                config.headers[SIGN_HDS[0]] = res.ts;
-                config.headers[SIGN_HDS[1]] = res.nonce;
-                config.headers[SIGN_HDS[2]] = res.version;
-                config.headers[SIGN_HDS[3]] = res.signature;
+
+                config.headers[SIGN_HDS[0]] = encodeURIComponent(res.ts);
+                config.headers[SIGN_HDS[1]] = encodeURIComponent(res.nonce);
+                config.headers[SIGN_HDS[2]] = encodeURIComponent(res.version);
+                config.headers[SIGN_HDS[3]] = encodeURIComponent(res.signature);
+                
                 return config;
             });
             this.request = rateLimit(request, { maxRPS });
