diff --git a/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js b/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
index eee6432..0cbc0c6 100644
--- a/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
+++ b/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
@@ -138,7 +138,7 @@ class TrezorKeyring extends events_1.EventEmitter {
             }
         });
         if (!this.trezorConnectInitiated) {
-            connect_web_1.default.init({ manifest: TREZOR_CONNECT_MANIFEST, lazyLoad: true });
+            connect_web_1.default.init({ manifest: TREZOR_CONNECT_MANIFEST, webusb: false, lazyLoad: true });
             this.trezorConnectInitiated = true;
         }
     }
