diff --git a/node_modules/trezor-connect/lib/popup/PopupManager.js b/node_modules/trezor-connect/lib/popup/PopupManager.js
index d794a8f..d6a8454 100644
--- a/node_modules/trezor-connect/lib/popup/PopupManager.js
+++ b/node_modules/trezor-connect/lib/popup/PopupManager.js
@@ -182,12 +182,16 @@ var PopupManager = /*#__PURE__*/function (_EventEmitter) {
             currentWindow: true,
             active: true
           }, function (tabs) {
-            _this4.extensionTabId = tabs[0].id; // $FlowIssue chrome not declared outside
+            var params = {
+              url: url
+            };

-            chrome.tabs.create({
-              url: url,
-              index: tabs[0].index + 1
-            }, function (tab) {
+            if (Array.isArray(tabs) && tabs.length > 0) {
+              _this4.extensionTabId = tabs[0].id;
+              params.index = tabs[0].index + 1;
+            } // $FlowIssue chrome not declared outside
+
+            chrome.tabs.create(params, function (tab) {
               _this4._window = tab;
             });
           });
